images/airflow_Dockerfile.sle_15sp1

```diff
@@ -0,0 +1,150 @@
+# Copyright 2018 AT&T Intellectual Property.  All other rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ---
+# (c) Copyright 2019 SUSE LLC
```

Comment:
I think the convention is to move line 15 to line 2, i.e., two separate lines of copy rights, but the Apache license is only needed once

Reply:
Fixed